pkg/machine/config.go

```diff
@@ -72,10 +72,10 @@ var (
 
 type Download struct {
```

Comment:
This never gets written to disk, right?

Reply:
correct, which is why it has no json stuff...